Packs/DragosWorldview/Integrations/DragosWorldview/README.md

```diff
@@ -0,0 +1,130 @@
+Custom integration designed to pull in reports from the Dragos Worldview API as incidents 
+This integration was integrated and tested with version 1.0 of Dragos Worldview 
+
+## Configure Dragos Worldview  on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Dragos Worldview .
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL (e.g. https://portal.dragos.com/api/v1/) |  | True |
```

Comment:
Please update the description according the change in usage the base url 
(using only https://portal.dragos.com/ etc)